quickstarts/zenduty/config.yml

```
@@ -0,0 +1,52 @@
+slug: zenduty
+
+# Used as the display name for the quickstart
+title: Zenduty
+
+description: |
+  ## What is Zenduty?
+  
+  Zenduty is a novel incident management platform providing end-to-end incident alerting, on-call management and response orchestration, giving teams greater control and automation over the incident management lifecycle.
+  
+  ### Why Zenduty?
+  
+  Zenduty provides your NOC, SRE and application engineers with detailed context around the New Relic APM alert along with playbooks and a complete incident command framework to triage, remediate and resolve incidents with speed.
+  
+  ### Get started!
+  
+  Check out the [Zenduty documentation](https://docs.zenduty.com/docs/newrelic) to learn how to setup best in class alerts around the New Relic APM.
+
+summary: |
+  Zenduty provides robust incident response, automation and notification abilities when you send your New Relic alerts to Zenduty. 
+
+# Possible values: New Relic | Verified | Community
+# Please consult with pull request reviewers if you think your quickstart should have a support level other than "Community"
+level: Community
+
+authors:
+  - Zenduty + Shubham Srivastava
```

Hey @zenbeam, this field accepts multiple entries. So we can just do this:
```suggestion
  - Zenduty
  - Shubham Srivastava
```